### PR TITLE
Builtins [echo + env + export]

### LINKED ISSUE #4: Implement builtins
As requested by the mandatory part of the subject, the following builtins must be implemented:

- [x] `echo`, with option `-n`
- [x] `cd`, with only a relative or an absolute path, and no options
- [x] `pwd`, with no options
- [x] `export`, with no options
- [x] `unset`, with no options
- [x] `env`, with no arguments, and no options
- [x] `exit`, with no options

### PR DESCRIPTION
**Pull request by JonathanDUFOUR**
The made changes are related to the issue https://github.com/Saphirac/minishell/issues/4.
The builtin `echo` has been implemented and tested.

### CODE REVIEW

**Comment by JonathanDUFOUR** [comment]
The `env` builtin has also been implemented and tested.

File: include/shell.h [hunk 1]
```diff
@@ -3,17 +3,17 @@
...
 /*   Created: 2023/03/08 20:36:31 by mcourtoi          #+#    #+#             */
-/*   Updated: 2023/03/09 15:18:33 by mcourtoi         ###   ########.fr       */
+/*   Updated: 2023/03/10 20:17:13 by jodufour         ###   ########.fr       */
 /*                                                                            */
 /* ************************************************************************** */
 
 #ifndef SHELL_H
 # define SHELL_H
 
-# include "minishell.h"
```
Comment (Saphirac) [inline include/shell.h]: Need to re-add minishell.h

**Review by Saphirac — CHANGES_REQUESTED** [review]
Warning : need to put back `minishell.h` instead of `list.h` inside shell.h

**Comment by JonathanDUFOUR** [comment]
The `export` builtin has also been implemented and tested.

File: include/minishell.h [hunk 2]
```diff
@@ -63,6 +63,9 @@ void	ft_free(char **tab);
 char	*access_path(char **paths, char *cmd);
 char	*add_path(char *str, char *av1, char c);
 char	*find_apath(char **env);
+int		surprise(void);
```
Comment (Saphirac) [inline include/minishell.h:66]: Approved.

**Review by Saphirac — APPROVED** [review]

**Pull request merged**